doc/guides/naming.md

```diff
@@ -41,530 +37,352 @@ The following are **singular**:
-        foreignKey: "opening_sales_person_id",
-      },
-    }, //👀
-  ],
-}
-```
+**API Implications:**
 
-**Database Implications**
+- This has no effect on the API.
 
-- Assumes a column `opening_sales_person_id` in the `account` table.
+**UI Implications:**
 
-**API Implications**
+- The `lastName` attribute will be displayed as "Surname" in the table header and filter dropdowns.
 
-This has no effect on the API.
+For more information on attributes: [attributes](../schema/attribute-types/README.md).
 
-### relationships.belongsToMany
+### relationships
 
-`target` must match a `Schema.name` and be _singular PascalCase_.
+#### belongsTo
 
-The following creates a belongsToMany relationship
-that acts similar to `hasMany`.
+`belongsTo` relationship fields should be singular camelCase.
 
-```js
-const SalesPerson = {
-  name: "SalesPerson",
+```ts
+const Account = {
+  name: "Account",
   attributes: {
-    firstName: "STRING",
+    name: string(),
   },
-  belongsToMany: [
-    {
-      target: "Account", //👀
-    },
-  ],
-}
+  relationships: {
+    salesPerson: belongsTo("SalesPerson"), //👀
+  },
+} satisfies PartialSchema
 ```
 
-**Database Implications**
+**Database Implications:**
 
-- Assumes a table `account_sales_person` exists with `sales_person_id` and `account_id` columns. The table is `account_sales_person` because account is first alphabetically.
+- Creates a column `sales_person_id` column in the `account` table.
 
-**API Implications**
+**API Implications:**
 
-- `accounts` will be available in the include query parameter like
-  `GET /api/sales-persons?include=accounts`
-- `accounts` will be used in mutation payloads and response payloads like:
+- `salesPerson` will be used in the `include` query parameter:
+  `GET /api/accounts?include=salesPerson`
+- `salesPerson` will be used in the mutation and response payloads:
   ```js
   {
     data: {
-      type: "SalesPerson",
-      id: "1",
-      attributes: { firstName: "Mary" },
+      type: "Account",
+      id: "2dba27ba-deaa-4656-a256-fb712f286c08",
+      attributes: { firstName: "Acme" },
       relationships: {
-        accounts: {
-          data: [{type: "Account", id: "456"}] //👀
+        salesPerson: { //👀
+          data: [
+            {
+              type: "SalesPerson",
+              id: "619d107b-0655-4a31-a704-13eb47d3f9c1"
+            }
+          ]
         }
       }
     }
   }
   ```
 
-### relationships.belongsToMany.options.through
-
-`through` must be _singular PascalCase_.
-
-```js
-const SalesPerson = {
-  name: "SalesPerson",
-  attributes: {
-    firstName: "STRING",
-  },
-  belongsToMany: [
-    {
-      target: "Account",
-      options: { through: "SalesAccount" },
-    },
-  ],
-}
-```
-
-**Database Implications**
+For more information on this relationhip type: [belongsTo](../schema/relationship-types/belongs-to.md).
 
-- Assumes a `sales_account` table exists with `sales_person_id` and `account_id` columns.
+#### hasMany
 
```

Comment:
did not document `hasMany.through()` since i don't think it has any implications on naming that isn't already covered here by `hasMany`, can you confirm @roy-coder @nlundquist

Reply:
I think the one naming related thing is the names of the columns used in the join table. It should also mention what the default names of those columns are by default and how to customize them imo?